atomic_reactor/plugins/koji_import.py

```diff
@@ -622,15 +624,18 @@ def _collect_remote_sources(self) -> Iterable[ArtifactOutputInfo]:
             dest_filename = remote_source_tarball['filename']
             yield local_filename, dest_filename, KOJI_BTYPE_REMOTE_SOURCES, None
 
-            remote_source_json = remote_source['remote_source_json']
-            remote_source_json_filename = remote_source_json['filename']
-            file_path = os.path.join(tmpdir, remote_source_json_filename)
-            with open(file_path, 'w') as f:
-                json.dump(remote_source_json['json'], f, indent=4, sort_keys=True)
-            yield (file_path,
-                   remote_source_json_filename,
-                   KOJI_BTYPE_REMOTE_SOURCES,
-                   None)
+            for source_key in (
```

Comment:
So each remote source has its own .config.json and .env.json files?

Reply:
yes